decidim-accountability/config/locales/en.yml

```diff
@@ -288,6 +288,23 @@ en:
         description: Number of results generated
         object: results
         title: Results
+    open_data:
```

Comment:
While reviewing, i actually got the following response to the accountability: 

```
### accountability

* id: The unique identifier of the result
* created_at: The date when the result was created
* body: Translation missing: en.decidim.open_data.help.accountability.body
* locale: Translation missing: en.decidim.open_data.help.accountability.locale
* author: Translation missing: en.decidim.open_data.help.accountability.author
* alignment: Translation missing: en.decidim.open_data.help.accountability.alignment
* depth: Translation missing: en.decidim.open_data.help.accountability.depth
* user_group: Translation missing: en.decidim.open_data.help.accountability.user_group
* commentable_id: Translation missing: en.decidim.open_data.help.accountability.commentable_id
* commentable_type: Translation missing: en.decidim.open_data.help.accountability.commentable_type
* root_commentable_url: Translation missing: en.decidim.open_data.help.accountability.root_commentable_url
```

Reply:
Fixed with 2912e9f88fc2c099ad93fd4886b6b6b5299c5dff